socorro/schemas/java_exception.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "$target_version": 2,
+  "type": "object",
+  "description": "JSON Schema for the JavaException crash annotation.",
+  "definitions": {},
+  "required": ["exception"],
+  "properties": {
+    "exception": {
+      "type": "object",
+      "required": ["values"],
+      "properties": {
+        "values": {
+          "type": "array",
+          "description": "Array of exception stacktraces",
+          "items": {
+            "type": "object",
+            "required": ["stacktrace"],
+            "properties": {
+              "stacktrace": {
+                "type": "object",
+                "description": "A single stacktrace",
+                "required": ["frames", "type", "module"],
```

Comment:
NullPointerException is an example of an exception that has no value. So we don't want to require "value" here.